otterdog/eclipse-jdtls.jsonnet

```diff
@@ -40,7 +40,7 @@ orgs.newOrg('eclipse.jdt.ls', 'eclipse-jdtls') {
     },
     orgs.newRepo('eclipse.jdt.ls') {
       allow_update_branch: false,
-      default_branch: "master",
+      default_branch: "main",
```

Comment:
this line can be removed completely, as the default is already "main", so when you dont set this property explicitly, the default will be taken.

Reply:
I think I noticed that as well (at https://github.com/EclipseFdn/otterdog-defaults/blob/main/otterdog-defaults.libsonnet#L24 ?), but when I saw we referenced it for branch protection rules I figured I'd keep it. Ok, makes sense to remove then.